apps/meteor/app/livechat/server/api/v1/config.ts

```diff
@@ -18,17 +19,17 @@ API.v1.addRoute(
 				return API.v1.success({ config: { enabled: false } });
 			}
 
-			const { token, department, businessUnit } = this.queryParams;
+			const { token, department, businessUnit } = this.queryParams as { token?: string; department?: string; businessUnit?: string };
 
 			const config = await cachedSettings({ businessUnit });
 
 			const status = Livechat.online(department);
-			const guest = token && (await Livechat.findGuest(token));
+			const guest: ILivechatVisitor | null = token ? await Livechat.findGuest(token) : null;
```

Comment:
I kinda remember live chat suffered from this because it doesn't trigger the default values, can we confirm this works?

Reply:
Sorry I didn't get this? Can you pls elaborate further?

Reply:
Checked it and this won't affect it